scripts/run_integration_tests.sh

```diff
@@ -1,91 +1,55 @@
 #!/usr/bin/env bash
 
-function get_frequency_pid () {
```

Comment:
Agree atleast for now we should merge check each conflict over accepting incoming/current changes

Reply:
I'll apply your changes. I was not sure which one was the latest since Wil changed the same file also. I'll apply ours and then if there are any disagreement can be handle when they are viewing the Stateful changes.